internal/envoy/v3/route.go

```diff
@@ -124,21 +124,37 @@ func RouteRoute(r *dag.Route) *envoy_route_v3.Route_Route {
 
 // hashPolicy returns a slice of hash policies iff at least one of the route's
 // clusters supplied uses the `Cookie` load balancing strategy.
```

Comment:
godoc can be updated for new impl (still references route's clusters, no `RequestHash` mentioned).